cirq-google/cirq_google/api/v2/program.proto

```diff
@@ -248,8 +248,47 @@ message NoiseChannel {
   }
 }
 
+// Representation of cirq_google.ops.AnalogDetuneCouplerOnly
+// All the following fields should be specified as the tunit.Value.
+message AnalogDetuneCouplerOnly{
+  Arg length= 1;  // The duration of gate 
```

Comment:
Should we specify that these are expected to be passed as tunits?

Reply:
I think i have stated that at the top of message:

// All the following fields should be specified as the tunit.Value.